CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
+## v1.2.1 (WIP)
+
+- Add logic to the empty GitHub post service. This will now post form contents to the github-provider.
+- Add refresh logic for the GitHub configurations.
```

Comment:
We've a bunch of conventions here that I haven't explained.

- One is that the changelogs shall start with one of the verbs from https://changelog.md, so suggest changing "Add" &rarr; "Added".

- Then since this is added features, SemVer2.0.0 states that this should be a minor version bump. Patch version bumps are for bug fixes. The "(WIP)" badge is spot on though!

- A PR reference should be added to the end. You can see lots of them have `!94` syntax, which is for GitLab. For GitHub we do `#8`.

- We apply `remark-lint` checks here, and one of the checks as Codacy states here is that it must be wrapped if it spans longer than 80 characters. You can run the linting locally by doing `npm i` and `npm run lint`

So all in all:

```suggestion
## v1.3.0 (WIP)

- Added logic to the empty GitHub post service. This will now post form
  contents to the github-provider. (#8)

- Added refresh logic for the GitHub configurations. (#8)
```